virttest/utils_conn.py

```diff
@@ -1762,9 +1763,20 @@ def __init__(self, *args, **dargs):
             'sasl_allowed_username_list', '["root/admin" ]')
         init_dict['traditional_mode'] = init_dict.get(
             'traditional_mode', 'no')
+        init_dict['run_on_remote'] = init_dict.get('run_on_remote', 'no')
 
         super(UNIXConnection, self).__init__(init_dict)
 
+        if init_dict['run_on_remote'] == 'yes':
+            self.client_ip = self.server_ip
```

Comment:
Do we really need override client_ip value under run_on_remote mode ? it looks like later logic still use self.server_ip to do relevant operations.

Reply:
This part was reworked in last update.